aiohttp/pytest_plugin.py

```diff
@@ -230,6 +230,23 @@ def proactor_loop():  # type: ignore
         yield _loop
 
 
+@pytest.fixture
+def selector_loop():  # type: ignore
+    if not PY_37:
+        policy = asyncio.get_event_loop_policy()
+        policy._loop_factory = asyncio.SelectorEventLoop  # type: ignore
```

Comment:
I don't think that it's wise to rely on private APIs.

Reply:
this is modified copypasta from `proactor_loop` :)